src/test/java/com/openshift/internal/restclient/model/v1/PodTest.java

```diff
@@ -81,13 +81,39 @@ public void testGetStatusTerminateReason() {
 		assertEquals(pod.getStatus(), "ReasonToTerminate");
 	}
 	
+	/**
+	 * Check that if both reason and exit code are set, status returns the reason.
+	 */
+	@Test
+	public void testGetStatusTerminateReasonAndExit() {
+		ModelNode node = new ModelNode();
```

Comment:
LGTM. In future, there is a ModelNodeBuilder class that might make it easier for your to build up the json.